### ISSUE TITLE
`idle_timeout_crazy_rtt` hangs more often now

### ISSUE THREAD

**Issue by larseggert** [mozilla/neqo#2162]
Something recently changed that makes the idle_timeout_crazy_rtt test not terminate pretty frequently, e.g., https://github.com/mozilla/neqo/actions/runs/11161582095/job/31024435251#step:7:1334

**Comment by martinthomson** [comment]
Your changes to reduce send rate when packets are lost will probably extend the time needed.

**Comment by larseggert** [comment]
But not by hours of real time?

**Comment by martinthomson** [comment]
No, not that.  I'm not seeing what conditions resulted in the stall though; the output of the run seems to have been lost.  If we have a seed for the test, it could be replicated locally.

**Comment by larseggert** [comment]
This one, for example: https://github.com/mozilla/neqo/actions/runs/11236671253/job/31237337009

**Comment by larseggert** [comment]
#2165 may help, once it's live in CI.

**Comment by larseggert** [comment]
This should not have closed, since #2165 wasn't a fix.

But it just found this seed, which caused a hang: `1856037d53afc27408ba15ff7b433b729f16e1ca087403a3e3eb9f2ed82856b5`.

**Comment by larseggert** [comment]
e2bca5650d71761da5ef7eac741b3880dbe8846bc7f8572a40371a065585605a failed here: https://github.com/mozilla/neqo/actions/runs/11888582176/job/33123413099?pr=2234

**Comment by larseggert** [comment]
I think this may be fixed? Have not seen this issue in a while.

**Comment by larseggert** [comment]
Closing this, since we haven't seen any recent occurrences.
